src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1399,6 +1399,11 @@ private bool OpenJsonFileFromPath(string filePath, bool forceManualExecutionMode
                         if (OpenJsonFile(filePath, fileContents, dynamoPreferences, out ws))
                         {
                             OpenWorkspace(ws);
+                            //Raise an event to deserialize the view parameters before
+                            //setting the graph to run
+                            OnComputeModelDeSerialized();
+
+                            //TODO: Can this be called from ViewModel instead? If so, OnComputeModelSerialized is not required. 
                             SetPeriodicEvaluation(ws);
```

Comment:
@mjkkirschner Can this call be moved to DynamoViewModel, after  `this.Model.OnWorkspaceOpening(viewInfo)` ? 
Is this kept here for running the graph without deserializing the view block?

Reply:
So, it's there for XMLOpening. Since XML deserialization works on the model, I think raising an event before causing the run should work fine.